projects/libgd/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./bootstrap.sh
+
+# Limit the size of buffer allocations to avoid bogus OOM issues
+# https://github.com/libgd/libgd/issues/422
+sed -i'' -e 's/INT_MAX/100000/' "$SRC/libgd/src/gd_security.c"
+
+./configure --prefix="$WORK" --disable-shared
+make -j$(nproc) install
+
+for target in Bmp Gif Tga Gd Gd2 WBMP; do
+    lowercase=$(echo $target | tr "[:upper:]" "[:lower:]")
+    $CXX $CXXFLAGS -std=c++11 -I"$WORK/include" -L"$WORK/lib" \
+      -DFUZZ_GD_FORMAT=$target \
+      $SRC/parser_target.cc -o $OUT/${lowercase}_target \
+      -lFuzzingEngine -lgd -lz
```

Comment:
zlib needs to be statically linked (grep the repo for `-Wl,-B` for the appropriate incantation